docs/standard/net-standard.md

```diff
@@ -13,20 +13,8 @@ ms.assetid: c044882c-af15-45f2-96d1-534557a5ee9b
 
 For information about choosing between .NET 5+ and .NET Standard, see [.NET 5+ and .NET Standard](#net-5-and-net-standard) later in this article.
 
-## .NET implementation support
-
-The various .NET implementations target specific versions of .NET Standard. Each .NET implementation version advertises the highest .NET Standard version it supports, a statement that means it also supports previous versions. For example, .NET Framework 4.6 implements .NET Standard 1.3, which means that it exposes all APIs defined in .NET Standard versions 1.0 through 1.3. Similarly, .NET Framework 4.6.1 implements .NET Standard 1.4, while .NET 5 implements .NET Standard 2.1.
-
-The following table lists the **minimum** implementation versions that support each .NET Standard version. That means that later versions of a listed implementation also support the corresponding .NET Standard version. For example, .NET Core 2.1 and later versions support .NET Standard 2.0 and earlier versions.
-
-[!INCLUDE [net-standard-table](../../includes/net-standard-table.md)]
```

Comment:
We should probably delete this include file as part of this PR. Or is it used elsewhere?

Reply:
Done